music_service_async_interface/__init__.py

```diff
@@ -176,35 +258,55 @@ class Cover(ABC):
 
     @abstractmethod
     def get_url(self, *args, **kwargs) -> str:
+        """Gets :class:`Cover` image URL
+
+        :param args: allows subfunction to define custom positional arguments
+        :param kwargs: allows subfunction to define custom keyword arguments
```

Comment:
Can you describe it better to me? What is a subfunction?

Reply:
I was forced to do this docs by linter, those are useless.
Subfunction is function overriding this function, but if you know better fitting name - tell me.

Reply:
`overriding function`?

Reply:
`implemented function`?